doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -238,16 +238,16 @@ Rewriting with Leibniz and setoid equality
    :name: rewrite *; _
    :undocumented:
 
-.. tacn:: replace @one_term__from with @one_term__to {? @occurrences } {? by @ltac_expr3 }
+.. tacn:: replace {? {| -> | <- } } @one_term__from with @one_term__to {? @occurrences } {? by @ltac_expr3 }
```

Comment:
```suggestion
.. tacn:: replace @one_term__from {| -> | <- | with } @one_term__to {? @occurrences } {? by @ltac_expr3 }
```

How about this syntax instead?  Then we can deprecate the current form and make the arrows and `with` work in the intuitive direction, a true fix for #13480.  WDYT?  This would also mean the most intuitive case (`->`) can be described first.

Maybe clearer to use nonterminal names `one_term__1` and `one_term__2` because to/from is misleading for some cases.

Reply:
This would make the syntax non-uniform with the other variants, though, and introduce a new kind of orientation argument. Rather, I think we should deprecate the version without orientation temporarily so that everybody uses `replace ->`, then switch the default behaviour and undeprecate the default.

Reply:
I would support that deprecation strategy so the bug is really fixed.

Reply:
Let's do that in another PR, in any case.